portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,18 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact() {
```

Comment:
Looks like you skipped a couple steps with the JS function (Making a servlet for your own endpoint (it's at /hello now I believe), calling your endpoint from your JS function, and modifying your servlet doGet() to be the one to serve the facts. (These are steps 1, 2, and 3 of the walkthroughs). I know you can do it this way, but just to practice working with the backend, please do it the way shown in the walkthroughs. If you're planning on doing these in later PR's, I would leave a TODO in the code here to let me (your reviewer) know what's going on. If you have any questions, please let me know!

Reply:
Resolved in second branch